service/policy/db/query.sql

```diff
@@ -27,3 +29,27 @@ RETURNING id;
 -- name: DeleteKeyAccessServer :execrows
 DELETE FROM key_access_servers WHERE id = $1;
 
+---------------------------------------------------------------- 
+-- RESOURCE MAPPING GROUPS
+----------------------------------------------------------------
+
+-- name: GetResourceMappingGroup :one
```

Comment:
We support LIST of all policy objects, so you'll probably need one here for groups as well.